src/shared/components/mutationMapper/MutationMapperDataStore.ts

```diff
@@ -13,22 +17,35 @@ import {Mutation} from "shared/api/generated/CBioPortalAPI";
 import {countDuplicateMutations, groupMutationsByGeneAndPatientAndProteinChange} from "shared/lib/MutationUtils";
 
 
-type CustomFilterApplier = (filter: DataFilter,
-                            mutation: Mutation,
-                            positions: {[position: string]: {position: number}}) => boolean;
+type GroupedData = {group: string, data: Mutation[][]}[];
 
-// TODO this is now mostly duplicate of DefaultMutationMapperDataStore in react-mutation-mapper, reuse DefaultMutationMapperDataStore instead
+export const PROTEIN_IMPACT_TYPE_FILTER_ID = "_cBioPortalProteinImpactTypeFilter_";
+export const MUTATION_STATUS_FILTER_ID = "_cBioPortalMutationStatusFilter_";
+
+export function findMutationTypeFilter(dataFilters: DataFilter[])
```

Comment:
function name sees strange because code finds a particular type of filter, right?  shouldn't function name suggest that?

Reply:
renamed to `findProteinImpactTypeFilter`